index.html

```diff
@@ -21,9 +21,8 @@
     <div class='header'></div>
 
     <div class='videoContainer'>
-        <div class='videoView'>
-            <video autoplay poster="static/img/webrtc-logo.png"/>
-        </div>
+        <video class='localVideo' autoplay poster="static/img/webrtc-logo.png"></video>
```

Comment:
good catch!

I say thanks for the mirrored image tip, my wife says thanks for muting the local stream 😆